### DISCUSSION TITLE
PWA icon per Codespace

### DISCUSSION THREAD

**Discussion by williamparry** [community/community#39730] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi,

Really enjoying Codespaces, so much so that I have 3 installed as PWAs on my Chromebook :)

However it would be handy to have an option to set an app icon for a repository (maybe we could upload under settings?) as at the moment my shelf looks like this:

![Screenshot 2022-11-22 20 38 06](https://user-images.githubusercontent.com/348091/203280889-fe0c5086-fae4-480b-b580-3fca266f9755.png)

Cheers!

**Comment by ossanna16** [comment]
Hi there @williamparry and welcome to our community! Thank you for the feedback!
